### PR TITLE
TFA_Support plugin update

### PR DESCRIPTION
**Pull request by Taxin2012**

### CODE REVIEW

**Comment by impulsh** [comment]
You seem to be missing content from your other pull request #62?

**Comment by Taxin2012** [comment]
Ye, you right, file updated

File: tfa_support/sh_tfa_support.lua [hunk 1]
```diff
@@ -24,6 +24,21 @@
...
+
+If `true`:
+	Auto-generate items for all weapons with `tfa_` prefix
+
+If `false`:
+	Auto-generate only that items that described in `sh_tfa_weps.lua` file
+
+*Black List works for both methods
+]]
+PLUGIN.DoAutoCreation = true
```
Comment (impulsh) [inline tfa_support/sh_tfa_support.lua]: You might want to document this info inside of `PLUGIN.readme` so people can read it on the site

Comment (Taxin2012) [reply tfa_support/sh_tfa_support.lua]: Updated

**Comment by Taxin2012** [comment]
Updated

**Pull request merged**